src/font/lv_font.h

```diff
@@ -39,6 +39,7 @@ typedef enum {
     /**< Legacy simple formats with no byte padding at end of the lines*/
     LV_FONT_GLYPH_FORMAT_A1     = 0x01, /**< 1 bit per pixel*/
     LV_FONT_GLYPH_FORMAT_A2     = 0x02, /**< 2 bit per pixel*/
+    LV_FONT_GLYPH_FORMAT_A3     = 0x03, /**< 3 bit per pixel*/
     LV_FONT_GLYPH_FORMAT_A4     = 0x04, /**< 4 bit per pixel*/
     LV_FONT_GLYPH_FORMAT_A8     = 0x08, /**< 8 bit per pixel*/
```

Comment:
@kisvegabor these don't get OR'd together anywhere, do they?

Reply:
Correct, it's not OR-ed.